wiki/Article_Styling_Criteria/en.md

```diff
@@ -877,61 +903,74 @@ Avoid: BNs can nominate your beatmapset.
 Do:    Members of the BN (Beatmap Nominators) can nominate your beatmapset.
 ```
 
-### Date formatting
+### Date and time
 
-Dates must be written in the [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) format, that is `YYYY-MM-DD`. For example:
+#### Date formatting
 
-- `27. November 2018` (`DD. MMMM YYYY`) must be `2018-11-27`
-- `April 1, 2017` (`MMMM dd, YYYY`) must be `2017-04-01`
-- `17 June, 2013` (`DD MMMM, YYYY`) must be `2013-06-17`
-- `2015 March, 02` (`YYYY MM, DD`) must be `2015-03-02`
-- `2009.09.09` (`YYYY.MM.DD` or `YYYY.DD.MM` and similar) must be `2009-09-09`
-- `2016年02月29日` must be `2016-02-29`
-- `2014년08월18일` must be `2014-08-18`
-- `2010年01月02日` must be `2010-01-02`
+When writing full dates in paragraphs, they must be formatted as `MMMM D, YYYY` (e.g. June 9, 2008); or where the year is omitted, must be `MMMM D` (e.g. July 28). The month and year must be fully written out, while the day is one-digit for days below 10.
```

Comment:
9 June is fine too

considering we follow wikipedia's thing exactly it might be better to link there https://en.wikipedia.org/w/index.php?title=Wikipedia:Manual_of_Style/Dates_and_numbers&oldid=925382361#Formats

Reply:
alright, wasn't sure if we just wanted to pick one and only one